### PR TITLE
CLOUDSTACK-10362: Change the "getXXX" method names to "isXXX".

### PR DESCRIPTION
**Pull request by BruceKuiLiu**
These Boolean-return methods are named "getXXX",
but other Boolean-return methods are named "isXXX", such as the following two methods.
They will return boolean values, rename them as "isXXX" should be more clear than "getXXX".

api/src/main/java/org/apache/cloudstack/api/command/admin/vlan/ListVlanIpRangesCmd.java
   ```
  public Boolean getForVirtualNetwork() {
         return forVirtualNetwork;
     }
```
api/src/main/java/org/apache/cloudstack/api/command/user/address/ListPublicIpAddressesCmd.java
   ```
public Boolean isForVirtualNetwork() {
        return forVirtualNetwork;
    }
```

### CODE REVIEW

File: api/src/main/java/org/apache/cloudstack/api/command/user/vm/DeployVMCmd.java [hunk 1]
```diff
@@ -256,7 +256,7 @@ public HypervisorType getHypervisor() {
         return HypervisorType.getType(hypervisor);
     }
 
-    public Boolean getDisplayVm() {
+    public Boolean isDisplayVm() {
```
Comment (DaanHoogland) [inline api/src/main/java/org/apache/cloudstack/api/command/user/vm/DeployVMCmd.java]: this obne is strange. it means to say shouldDisplayVm() but is can live with it

File: api/src/main/java/org/apache/cloudstack/api/command/user/vpc/UpdateVPCCmd.java [hunk 2]
```diff
@@ -73,7 +73,7 @@ public Long getId() {
         return id;
     }
 
-    public Boolean getDisplayVpc() {
+    public Boolean isDisplayVpc() {
```
Comment (DaanHoogland) [inline api/src/main/java/org/apache/cloudstack/api/command/user/vpc/UpdateVPCCmd.java]: same here; shouldDisplayVpc()??

File: api/src/main/java/org/apache/cloudstack/api/response/HostResponse.java [hunk 3]
```diff
@@ -646,11 +646,11 @@ public String getEvents() {
         return events;
     }
 
-    public Boolean getHasEnoughCapacity() {
+    public Boolean hasEnoughCapacity() {
```
Comment (DaanHoogland) [inline api/src/main/java/org/apache/cloudstack/api/response/HostResponse.java]: good choice

File: api/src/main/java/org/apache/cloudstack/api/response/ServiceOfferingResponse.java [hunk 4]
```diff
@@ -180,7 +180,7 @@ public void setIsSystemOffering(Boolean isSystem) {
         this.isSystem = isSystem;
     }
 
-    public Boolean getDefaultUse() {
+    public Boolean isDefaultUse() {
```
Comment (DaanHoogland) [inline api/src/main/java/org/apache/cloudstack/api/response/ServiceOfferingResponse.java]: isForDefaultUse() ??!?

File: plugins/database/quota/src/main/java/org/apache/cloudstack/api/command/QuotaSummaryCmd.java [hunk 5]
```diff
@@ -94,7 +94,7 @@ public String getCommandName() {
         return s_name;
     }
 
-    public Boolean getListAll() {
+    public Boolean isListAll() {
```
Comment (DaanHoogland) [inline plugins/database/quota/src/main/java/org/apache/cloudstack/api/command/QuotaSummaryCmd.java]: shouldListAll()???

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan test

**Comment by DaanHoogland** [comment]
these test resulta are dissappointing so i did a recursive case insensitive search of the changed method names:
for i in getOfferHa,isOfferHa,GetLimitCpuUse,isLimitCpuUse,getVolatileVm,isVolatileVm,getForVirtualNetwork,isForVirtualNetwork,getIsSourceNat,isSourceNat,getIsStaticNat,isStaticNat,getRestartRequired,isRestartRequired,getSpecifyIpRanges,isSpecifyIpRanges,getDisplayVm,isDisplayVm,getDisplayVpc,isDisplayVpc,getLocalStorageActive,isLocalStorageActive,getHasEnoughCapacity,hasEnoughCapacity,getSuitableForMigration,isSuitableForMigration,getDefaultUse,isDefaultUse,getForDisplay,isForDisplay,getDestroyed,isDestroyed,getListAll,isListAll,getSparseVolumes,isSparseVolumes,getPasswordEnabled,isPasswordEnabled,getRequiresHVM,isRequiresHVM,getIsPublic,isPublic,getFeatured,isFeatured,getIsExtractable,isExtractable,getBootable,isBootable,getSshKeyEnabled,isSshKeyEnabled
do
  grep -ir $i *
done
```
baloujak:pr2600-t2589-kvm-centos7 dahn$ for i in getOfferHa,isOfferHa,GetLimitCpuUse,isLimitCpuUse,getVolatileVm,isVolatileVm,getForVirtualNetwork,isForVirtualNetwork,getIsSourceNat,isSourceNat,getIsStaticNat,isStaticNat,getRestartRequired,isRestartRequired,getSpecifyIpRanges,isSpecifyIpRanges,getDisplayVm,isDisplayVm,getDisplayVpc,isDisplayVpc,getLocalStorageActive,isLocalStorageActive,getHasEnoughCapacity,hasEnoughCapacity,getSuitableForMigration,isSuitableForMigration,getDefaultUse,isDefaultUse,getForDisplay,isForDisplay,getDestroyed,isDestroyed,getListAll,isListAll,getSparseVolumes,isSparseVolumes,getPasswordEnabled,isPasswordEnabled,getRequiresHVM,isRequiresHVM,getIsPublic,isPublic,getFeatured,isFeatured,getIsExtractable,isExtractable,getBootable,isBootable,getSshKeyEnabled,isSshKeyEnabled
> do
>   grep -ir $i *
> done
baloujak:pr2600-t2589-kvm-centos7 dahn$
```

no results are good results. I think we can go agead with this @rhtyd @rafaelweingartner ?

**Comment by rafaelweingartner** [comment]
@DaanHoogland so you checked it, and it seems that everything is ok? Is that it?
If that is the case, I think we proceed as we did with others. Travis and Jenkins are green.

**Pull request merged**